packages/plugin-ebarimt-api/src/graphql/resolvers/queries/productGroups.ts

```
@@ -0,0 +1,95 @@
+      defaultValue: []
+    });
+    const productIds = products.map(p => p._id);
+
+    filter.$or = [
+      { mainProductId: { $in: productIds } },
+      { subProductId: { $in: productIds } }
+    ]
+  }
+
+  if (params.status) {
+    if (params.status = 'active') {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)